core/src/main/java/hudson/security/TokenBasedRememberMeServices2.java

```diff
@@ -108,11 +112,6 @@ protected String makeTokenSignature(long tokenExpiryTime, String username, Strin
         return MAC.mac(token);
     }
 
-    @Override
-    protected String retrievePassword(Authentication successfulAuthentication) {
-        return "N/A";
-    }
```

Comment:
This was just being overridden to satisfy the contract of `TokenBasedRememberMeServices`, but it was never used (and indeed could have thrown `UnsupportedOperationException`). Now that the parent is `AbstractRememberMeServices` it is no longer needed.